src/ace.F90

```diff
@@ -418,6 +423,68 @@ subroutine read_ace_table(i_table, i_listing)
   end subroutine read_ace_table
 
 !===============================================================================
+! READ_MULTIPOLE_DATA checks for the existence of a multipole library in the
+! directory and loads it using multipole_read
+!===============================================================================
+
+  subroutine read_multipole_data(i_table)
+
+    integer, intent(in) :: i_table   ! index in nuclides/sab_tables
+
+    logical       :: file_exists   ! does multipole library exist?
+    character(7)  :: readable      ! is multipole library readable?
+    character(6)  :: zaid_string   ! String of the ZAID
+    character(MAX_FILE_LEN+9)  :: filename  ! path to multipole xs library
```

Comment:
`path_multipole` is a `character(MAX_FILE_LEN)` so adding the `+9` (for ZZZAAA.h5) avoids the character truncation warning.